### ISSUE TITLE
Test report - success story (?)

### ISSUE THREAD

**Issue by hustf** [JuliaGL/GLVisualize.jl#100]
Thanks for the continued great effort. I had a go at checking the current state of GLVisualize, and hope these details can save some time:

Julia 0.4.6
Windows 10 64 bit, OK graphics card.
GLVisualize 0.0.9+
GLAbstraction 0.0.13+ 
BilliardModels 0.0.0- time_step (unregistered) - for one of the examples.
FixedSizeArrays 0.2.1 
GeometryTypes 0.1.5+
GLWindow 0.1.3+
Reactive  0.3.3+

The cairo library created difficulties as usual, which was fixed with: BinDeps.debug("Cairo")

```
julia> cd(Pkg.dir("GLVisualize","test"))

julia> include("runtests.jl")
WARNING: similar{FSA<:FixedArray}(::Type{FSA}, ...) is deprecated, use similar_type instead
 in depwarn at deprecated.jl:73
 in gl_convert at C:\Users\F\.julia\v0.4\GLAbstraction\src\GLUniforms.jl:212
(snip)
while loading C:\Users\F\.julia\v0.4\GLVisualize\test\runtests.jl, in expression starting on line 63
ERROR: LoadError: ArgumentError: No built in FixedArray type is implemented for eltype Float32         and size ()
 in similar_type at C:\Users\F\.julia\v0.4\FixedSizeArrays\src\core.jl:98
while loading C:\Users\F\.julia\v0.4\GLVisualize\test\runtests.jl, in expression starting on line 63
```

Also checking out FixedSizeArrays 0.2.1+ and restarting Julia:

```
julia>
cd(Pkg.dir("GLVisualize","test"))
julia> include("runtests.jl")
Neglected warning:
FixedSizeArrays.jl:9 WARNING: could not import LinAlg._chol! into FixedSizeArrays

Error message at runtests.jl:63
ERROR: LoadError: ArgumentError: No built in FixedArray type is implemented for eltype Float32 and size()
in similar_type at FixedSizeArrays\src\core.jl:98
```

**Comment by SimonDanisch** [comment]
Thanks @hustf !
I actually just finally found the time to fix these :)
Let me put up a PR and tag this!

**Comment by SimonDanisch** [comment]
Btw, a temporary fix is to checkout  FixedSizeArrays `v"0.1.2"`. Note that the package manager seems to be very eager, to set the version of FixedSizeArrays back to 2.1 whenever you run the tests. Not sure what the story there is.

**Comment by hustf** [comment]
The issue response time here is certainly faster than the package manager!
Pkg.pin("FixedSizeArrays",v"0.1.2")  fixes this. That's success story enough. Thank you!

Those nodding cats are quite funny, and the mouse wheel changes lens focus. Superb! 

So far, the version stays pinned, although I expect trouble when using other packages. I have been advised to use Virtual Machines to avoid that kind of thing. In coming Julia versions we may get something similar to Python Anaconda's environments. Until then I suppose using the REQUIRE file to specify all the indirectly referred packages is best practice.
